wscript

```diff
@@ -399,6 +405,41 @@ def collect_dirs_to_recurse(bld, globs, **kw):
 def list_boards(ctx):
     print(*boards.get_boards_names())
 
+def list_ap_periph_boards(ctx):
+    print(*boards.get_ap_periph_boards())
+
+@conf
+def ap_periph_boards(ctx):
+    return boards.get_ap_periph_boards()
+
+def generate_tasklist(ctx, do_print=True):
+    boardlist = boards.get_boards_names()
+    ap_periph_targets = boards.get_ap_periph_boards()
+    tasks = []
+    with open(os.path.join(Context.top_dir, "tasklist.json"), "w") as tlist:
+        for board in boardlist:
+            task = {}
+            task['configure'] = board
+            if board in ap_periph_targets:
+                if 'sitl' not in board:
+                    # we only support AP_Periph and bootloader builds
+                    task['targets'] = ['AP_Periph', 'bootloader']
+                else:
+                    task['targets'] = ['AP_Periph']
+            elif 'iofirmware' in board:
+                task['targets'] = ['iofirmware', 'bootloader']
+            else:
+                if 'linux' in board:
+                    # we only support AP_Periph and bootloader builds
+                    task['targets'] = ['antennatracker', 'copter', 'heli', 'plane', 'rover', 'sub', 'bootloader', 'replay']
```

Comment:
replay is not linux any more, it's normally sitl but can do any board. Maybe list as only sitl